index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-First</title>
+
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/nav.css">
+  <link rel="stylesheet" href="styles/info.css">
+  <link rel="stylesheet" href="styles/footer.css">
+</head>
+<body>
+  <main>
+    <header id="header">
+      <div class="header-container">
+        <a href="https://twitter.com/black_okyere"><p>Skabeo</p></a>
```

Comment:
- [OOTIONAL] The logo should be a link that links back to the home page. You link leads to twitter 🐦